docs/sdk/client-side/javascript/index.md

```diff
@@ -89,38 +94,51 @@ The Bucketeer SDK doesn't save the user data. The Application must save and set
 Initialize the client by passing the configurations in the previous step.
 
 <Tabs>
-<TabItem value="js" label="Javascript">
+<TabItem value="js" label="JavaScript">
 
 ```js showLineNumbers
-const client = BKTClient.initialize(config, user);
+await initializeBKTClient(config, user);
```

Comment:
Just confirming. Can the the init process be implemented like this?

```js
const client = await BKTClient.initialize(config, user);
```

Reply:
I see.

I prefer the way you suggested.
Is it going to take too much time to change it?

Reply:
done: https://github.com/bucketeer-io/javascript-client-sdk/pull/56

Reply:
You are right.
I also checked the iOS SDK, and the initialization is made in the same way as Android SDK.

https://github.com/ca-dp/bucketeer-ios-sdk/blob/v2/Bucketeer/Sources/Public/BKTClient.swift#L76-L95

I didn't like to have to import these functions `initializeBKTClient` and `getBKTClient`, but since the Typescript doesn't support static members in the interface, I think there is nothing to do about it.

I was reading this [topic](https://stackoverflow.com/questions/13955157/how-to-define-static-property-in-typescript-interface), but not sure that would work for us.

If there is no easy workaround for the static members, we should keep the current implementation consistent with the other SDKs.

Please let me know your thoughts.

Reply:
I think JavaScript libraries tend to work this way(import what you need on demand).

here's the example 
https://firebase.google.com/docs/web/setup?hl=en

The old bucketeer SDK uses wildcard import but that's not a recommended as it imports everything from the target, and that increases bundle size.

(By the way if you prefer you can write like that)

```ts
import * as BKTClient from '@bucketeer/javascript-client-sdk'

await BKTClient.initializeBKTClient(...)
```

Reply:
I see.

Is it possible to rename the `initializeBKTClient` to `initialize` and use it as the following? Is there any concern about doing this way?

```typescript
import * as BKTClient from '@bucketeer/sdk'
await BKTClient.initialize(...)
```